packages/react-lowcode/src/templates/index.ts

```diff
@@ -0,0 +1,49 @@
+export const entityFieldMuiMap = `
+(eF) => {
+  return {
+    field: eF.name,
+    flex: 1,
+    type: eF.type,
+    valueFormatter: getValueFormatterFunction(eF.type),
+    renderHeader: (params: GridColumnHeaderParams) => (
+      <FormattedMessage id={\`\${eF.name}.\${eF.type}\`} defaultMessage={eF.name} />)
```

Comment:
`<FormattedMessage` is ideally behind a template/function

id={\`\${lowerCamelCase(eF.entityName)}.\${lowerCamelCase(eF.fieldName)}\`}